ironfish/src/storage/database/transaction.ts

```diff
@@ -18,6 +22,8 @@
  * or write is performed on it.
  */
 export interface IDatabaseTransaction {
+  cache: BufferMap<unknown>
```

Comment:
Should this be a generic?

Reply:
updated to be a generic `Map<Buffer, unknown>` interface